src/lib/store.js

```diff
@@ -33,6 +33,9 @@ const initialState = {
   isSideNavExpanded: false,
   isSearchExpanded: false,
 
+  // Whether to show the progressbar and mark the main content as busy, during a load.
+  duringPageLoad: false,
```

Comment:
Elsewhere I've been using an `is` prefix for boolean state stuff like `isSearchExpanded`.
wdyt about renaming to `isPageLoading` ?

Reply:
Done